cypress/integration/home.spec.ts

```diff
@@ -1,3 +1,40 @@
+it('should show correct page title', () => {
+  cy.visit('/')
+
+  cy.title().should(
+    'equal',
+    'Homepage | Octoclairvoyant: Compare GitHub changelogs across multiple releases'
+  )
+})
+
+it('should display h1 heading with correct text', () => {
+  cy.visit('/')
+
+  cy.findByRole('heading', { level: 1, name: 'Octoclairvoyant' }).should(
+    'exist'
+  )
+})
+
+it('should display h2 heading with correct text', () => {
+  cy.visit('/')
+
+  cy.findByRole('heading', {
+    level: 2,
+    name: 'Compare GitHub changelogs across multiple releases',
+  }).should('exist')
+})
+
+it('should display h3 heading with correct text', () => {
+  cy.visit('/')
+
+  cy.findByRole('heading', {
+    level: 3,
+    name: 'Features',
+  }).should('exist')
+})
```

Comment:
It's better to wrap all this in a single test. The philosophy should be trying to wrap similar assertions under the same test, so these can be grouped in a `"should display corresponding information"` test where you visit the home page just once and then assert expected elements.

Creating a test for each one of those assertions will make our test execution really slow since Cypress has to tear down and set up the tests again, visiting the home page over and over.

Reply:
Wrapped most of the tests under a single test now. Should be faster to run now. 😄